test/integration/organizations/staff/adoption_application_reviews_test.rb

```diff
@@ -24,6 +30,18 @@ class Organizations::Staff::AdoptionApplicationReviewsTest < ActionDispatch::Int
       follow_redirect!
       assert_equal I18n.t("errors.authorization_error"), flash[:alert]
     end
+
+    should "not be able to view application details" do
+      @user = create(:user)
+      sign_in @user
+
+      get staff_adoption_application_review_path(@adopter_application)
```

Comment:
I wonder if this is actually failing because of `Organizations::AdopterApplicationPolicy` `manage?` method, or in the pre-check for `verify_active_staff!`